server/api.js

```diff
@@ -356,6 +356,71 @@ router.patch("/v1/processes/:ref/appraise", (req, res) => {
   proxy(options, req, res);
 });
 
+router.post("/v1/processes/:ref/post-visit-actions", (req, res) => {
+  if (!verifyJwt(req.query.jwt, matApiJwtSecret)) {
+    res.status(401).send("Invalid JWT");
+
+    return;
+  }
+
+  const { ref } = req.params;
+  const { description, category, subcategory, data } = req.body;
+
+  const {
+    matApiToken,
+    areaId,
+    contactId,
+    subjectId,
+    patchId,
+    householdId,
+    officerFullName,
+    officerId,
+  } = decryptJson(
+    data,
+    matApiDataSharedKey,
+    matApiDataSalt,
+    matApiDataIterations,
+    matApiDataKeyLength,
+    matApiDataAlgorithm,
+    matApiDataIV
+  );
+
+  const options = {
+    host: matApiHost,
+    port: 443,
+    path: `${matApiBaseUrl}/v1/TenancyManagementInteractions/CreateTenancyManagementInteraction`,
+    method: req.method,
+    headers: {
+      Authorization: matApiToken,
+      "Content-Type": "application/json",
+    },
+    timeout: 10 * 1000,
+  };
+
+  req.body = {
+    subject: subjectId,
+    source: "1", //leave as 1, maps to "Estate officer" value for hackney_handleby filed
+    processType: "2", //Post visit action
+    ServiceRequest: {
+      title: "Tenancy Management", //Any title
```

Comment:
Is this is any title, let's use the one from the env: `process.env.PROCESS_TYPE_NAME`.